stories/tour-tooltips.stories.js

```diff
@@ -0,0 +1,369 @@
+												href="/wordpress-websites/wordpress-programming/"
+												target="_blank"
+												rel="noopener noreferrer"
+												aria-label="/wordpress-websites/wordpress-programming/ (opens in a new tab)"
+											>
+												/wordpress-websites/wordpress-programming/
+											</a>
+										</div>
+									</td>
+									<td className="googlesitekit-table__body-item">
+										<div className="googlesitekit-table__body-item-content">
+											27
+										</div>
+									</td>
+								</tr>
+								<tr className="googlesitekit-table__body-row">
+									<td className="googlesitekit-table__body-item">
+										<div className="googlesitekit-table__body-item-content">
+											<a
+												className="googlesitekit-cta-link googlesitekit-table__body-item-link googlesitekit-cta-link--inherit"
+												href="https://earthbound.com/wp-admin/admin.php?page=googlesitekit-dashboard&amp;permaLink=https%3A%2F%2Fearthbound.com%2Fcategory%2Fwordpress%2F"
+											>
+												WordPress – Earthbound
+											</a>
+											<a
+												className="googlesitekit-cta-link googlesitekit-table__body-item-url googlesitekit-cta-link--external googlesitekit-cta-link--inherit"
+												href="/category/wordpress/"
+												target="_blank"
+												rel="noopener noreferrer"
+												aria-label="/category/wordpress/ (opens in a new tab)"
+											>
+												/category/wordpress/
+											</a>
+										</div>
+									</td>
+									<td className="googlesitekit-table__body-item">
+										<div className="googlesitekit-table__body-item-content">
+											20
+										</div>
+									</td>
+								</tr>
+							</tbody>
+						</table>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+);
+
+storiesOf( 'Global', module )
+	.add( 'TourTooltips', () => {
+		const steps = [
+			{
+				target: '.step-1',
+				title: 'See where your visitors are coming from',
+				content: 'Click on a slice of the chart to see how it changed over time just for that source',
+				placement: 'bottom-start',
+			},
+			{
+				target: '.step-2',
+				title: 'It\'s now easier to see your site\'s traffic at a glance',
+				content: 'Check the trend graph to see how your traffic changed over time',
+				placement: 'top-end',
+			},
+			{
+				target: '.step-3',
+				title: 'Check how your traffic changed since you last looked',
+				content: 'Select a time frame to see the comparison with the previous time period',
+			},
+			{
+				target: '.step-4',
+				title: 'Generic step title for the fourth step',
+				content: (
+					<div>
+						This is the fourth step with an external link that should go to Google, <Link href="http://google.com" external>link.</Link>
+					</div>
+				),
+			},
+		];
+
+		return (
+			<WithTestRegistry>
+				<MockWPDashboard />
+				<TourTooltips steps={ steps } tourID="feature" />
```

Comment:
@eugene-manuilov While making this change, I realized it would remove our ability to verify conditional rendering within the QA Brief. I'm on the fence — using a static `tourID` allows us to test the component using expected functionality within Storybook, but could potentially confuse an engineer if they had already cleared the tour, forgot, then tested within Storybook again at a later date.

What do you think?

Reply:
Ok, let's leave it as is for now. It will be updated in the #2650.